docs/docs/widgets/image.md

```diff
@@ -34,59 +34,59 @@ Check [Action Reference](/docs/category/actions-reference) docs to get the detai
 
```

Comment:
For line no. 32
 you have made a different change for this particular line in the LTS version, please check it and make the constant changes in both the docs.